x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/actions_popover.test.tsx

```diff
@@ -119,14 +119,33 @@ describe('ActionsPopover', () => {
       />
     );
 
-    expect(getByRole('link')?.getAttribute('href')).toBe('/a/test/edit/url');
+    expect(getByTestId('editMonitorLink')?.getAttribute('href')).toBe('/a/test/edit/url');
+  });
+
+  it('contains link to clone monitor', async () => {
+    jest
+      .spyOn(editMonitorLocatorModule, 'useEditMonitorLocator')
+      .mockReturnValue('/a/test/edit/url');
+    const { getByTestId } = render(
```

Comment:
We could also use `getByText` here to rely on a more accessible identifier for the element.

Reply:
actually it can't be, i need to check for href prop, which is bit above the text in hierarcy

Reply:
Ah interesting! Doing `getByText('Clone monitor')` won't pick it up?